lib/passwords.ts

```diff
@@ -192,6 +210,16 @@ export class Passwords {
     });
   }
 
+  resetByExistingPassword(
+    data: ResetByExistingPasswordRequest
```

Comment:
Can we add the email, newPassword, and oldPassword as required fields like the resetByEmail method?